pkg/services/ngalert/notifier/external_alertmanager.go

```diff
@@ -121,30 +123,62 @@ func (am *externalAlertmanager) ListSilences(ctx context.Context, filter []strin
 	return res.Payload, nil
 }
 
-func (am *externalAlertmanager) GetStatus() apimodels.GettableStatus {
-	return apimodels.GettableStatus{}
+func (am *externalAlertmanager) GetAlerts(ctx context.Context, active, silenced, inhibited bool, filter []string, receiver string) (apimodels.GettableAlerts, error) {
```

Comment:
Should add a recover here as `GetAlerts` can panic.

Reply:
Where exactly can it panic? The `res.Payload` bit?

Reply:
`GetAlerts(params)` panics